web/src/compose_ui.ts

```diff
@@ -104,7 +122,7 @@ export function smart_insert_inline($textarea, syntax) {
     // start of the content after the syntax, add one.
     if (
         !(
-            (after_str.length > 0 && is_space(after_str[0])) ||
+            (after_str && after_str.length > 0 && is_space(after_str[0])) ??
```

Comment:
Why is this changed to `??~? I'm not sure why it'd be correct to avoid evaluating the second half; but I guess in any case `after_str` is not intended to be possibly undefined here.